english/NarcosStory/narcos.md

```diff
@@ -0,0 +1,8 @@
+Narcos (Imdb 8.9)
+
+Narcos is an American crime web television series created and produced by Chris Brancato, 
+Carlo Bernard, and Doug Miro. Season 1, comprising 10 episodes, originally aired on August 28, 2015, as a Netflix exclusive.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.